### DISCUSSION TITLE
My school does not provide student emails.

### DISCUSSION THREAD

**Discussion by olencer** [community/community#72736] [GitHub Education]
### Hi everyone,

My college only offers the edu email to the staff, I can't get one. 
My request for student benefits has been rejected, and it asked for an academic email address.
![1](https://github.com/community/community/assets/47381894/d2915536-d9ac-4b5a-9512-e783783cca65)

**Comment by PatilSpeaks** [comment]
Submit the college fee receipt, which shows that you are currently studying in that course.

**Comment by ZainAsif767** [comment]
I am also facing the same issue as my university doesn't offer school emails. I managed to use my email address for the first four attempts, but my application was rejected on the fifth try, stating that I needed to use an email provided by the university.
